Library/Homebrew/cask/lib/hbc/artifact/binary.rb

```diff
@@ -9,7 +9,7 @@ def install_phase
 
       def link
         super
-        FileUtils.chmod "+x", source
+        @command.run!("/bin/chmod", args: ["+x", source])
```

Comment:
Sorry, I missed the notification. I probably should have mentioned in the HBC issue that I don't actually know ruby (or programming in general). I'm not sure how to skip the command if it is already executable.
```
  if source.writable?
    FileUtils.chmod "+x", source
  else
    SystemCommand.run("/bin/chmod", args: ["+x", source], sudo: true))
  end
```